frontend/components/MDM/SSOError/SSOError.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+import DataError from "components/DataError";
+
+const baseClass = "mdm-sso-error";
+
+const SSOError = () => {
+  return (
+    <DataError className={baseClass}>
+      <p>Please contact your IT admin at +1-(415)-651-2575.</p>
+    </DataError>
+  );
+};
```

Comment:
Looks good. I'd recommend adding `className` optional prop to allow the parent to put a custom class on this. Our common components tend to add this just in case some overrides are needed that don't make sense to add as a prop here.

the pattern we tend to use is something like this

```tsx
interface ISSOErrorProps {
  className?: string;
}

const SSOError = ({className}: ISSOErrorProps) => {
  const classeNames = classnames(baseClass, className)

  return (
    <DataError className={classNames}>
      <p>Please contact your IT admin at +1-(415)-651-2575.</p>
    </DataError>
  );
};
```

Reply:
sounds good! I pushed a fix

question: why are we doing this preemptively? none of the callers need to pass a class now, why don't add this on a per-needed basis?

Reply:
we could do either but been trying to get into a pattern to always do this for generic components. Its not critical though

Reply:
gotcha, thanks! I added the class, we'll be touching this component soon, I hope it comes handy!